patches/minecraft/net/minecraft/server/dedicated/ServerHangWatchdog.java.patch

```diff
@@ -8,16 +8,26 @@
  
      public ServerHangWatchdog(DedicatedServer p_i46310_1_)
      {
-@@ -37,7 +38,7 @@
+@@ -37,14 +38,15 @@
              long j = MinecraftServer.func_130071_aq();
              long k = j - i;
  
 -            if (k > this.field_180250_c)
 +            if (k > this.field_180250_c && !this.firstRun)
              {
-                 field_180251_a.fatal("A single server tick took {} seconds (should be max {})", String.format("%.2f", (float)k / 1000.0F), String.format("%.2f", 0.05F));
+-                field_180251_a.fatal("A single server tick took {} seconds (should be max {})", String.format("%.2f", (float)k / 1000.0F), String.format("%.2f", 0.05F));
++                org.apache.logging.log4j.message.Message message = field_180251_a.getMessageFactory().newMessage("ServerHangWatchdog detected that a single server tick took {} seconds (should be max {})", String.format("%.2f", (float)k / 1000.0F), String.format("%.2f", 0.05F)); // Forge: clarify where this is coming from, and save it to put in the Error too
```

Comment:
What you are doing here is good coding practice but it's not very good for patches, please just add your improved message to the Error so that this is a small change.

Reply:
Done.